tools/test_api.py

```diff
@@ -1963,33 +1964,18 @@ def test_path_to_name(path):
     
     return "-".join(name_parts).lower()
 
-def find_tests(base_dir):
+def find_tests(base_dir, target, toolchain, options):
```

Comment:
This is so tests can be located under `TARGET_DEBUG` and `TARGET_RELEASE` folders